transforms/angle-brackets/transform.js

```diff
@@ -280,6 +280,21 @@ function hasValuelessDataParams(params) {
   return getDataAttributesFromParams(params).length > 0;
 }
 
+/**
+ *
+ * data-* attributes are generally omitted,
+ * but this config allows including nodes with data-test-* attributes.
+ */
+function shouldSkipDataTestParams(params, includeValuelessDataTestAttributes) {
+  if (includeValuelessDataTestAttributes) {
+    const dataAttrs = getDataAttributesFromParams(params);
+    // This is true for nodes with data-* attributes too,
+    // as long as there is one with data-test-* attribute.
+    return !dataAttrs.some(attr => attr.original.startsWith('data-test'));
```

Comment:
just to be safe it should be `.startsWith('data-test-')` so that it doesnt match cases like `data-testfoo`